services/bls_aggregation/blsagg.go

```diff
@@ -350,8 +344,28 @@ func (a *BlsAggregatorService) singleTaskAggregatorGoroutineFunc(
 				// happens..
 				continue
 			}
-			// after verifying signature we aggregate its sig and pubkey, and update the signed stake amount
+
+			// check if the operator has already signed for this digest
 			digestAggregatedOperators, ok := aggregatedOperatorsDict[taskResponseDigest]
+			if ok {
+				if digestAggregatedOperators.signersOperatorIdsSet[signedTaskResponseDigest.OperatorId] {
+					a.logger.Warn(
```

Comment:
```suggestion
					a.logger.Info(
```

Reply:
Let's keep warning for things that shouldn't happen but that we should pay attention to, see https://stackoverflow.com/a/2031209/4971151

duplicate signature isn't interesting, unless we receive 10k of them say

Reply:
https://github.com/Layr-Labs/eigensdk-go/pull/345/commits/1fa0bb48927c69a7557df88627248062d6cfaa4f maybe adaptive logging is something we should consider as an enhancement down the line, there's probably a few places it would be useful